packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -125,10 +124,24 @@
                 data-ng-model="$ctrl.model.name"
                 data-ng-pattern="$ctrl.NAME_PATTERN"
                 data-ng-maxlength="$ctrl.MAX_NAME_LENGTH"
+                data-ng-model-options="{ allowInvalid: true }"
                 required
             />
+            <div class="oui-password__rules" id="database-name-rules">
```

Comment:
Following class `oui-password__rules` is meant to be used by the given component [`<oui-password>`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-password-webcomponent--password-rules-strength).

see: https://github.com/ovh/ovh-ui-kit/blob/8015345cd92efd473c18fcf68aa28d91fe278e78/packages/components/password/src/js/password.html#L43

**question:** Why applying this class for a regular text input?

Reply:
Hi Antoine,
The point of this PR is to be able to have the same visual as `oui-password__rules` for any input.  We discussed it with Marie (here: https://github.com/ovh/manager/pull/6263#discussion_r782804286), existing components in the ovh-ui-kit does not fit the need, hence why I created the component.
It has been mentioned multiple times in previous PR that writhing custom css should be avoided, hence why I used the existing class to have the correct visual.

Reply:
Guidelines around input elements needs to be updated then (including components).
Applying an helper from `<oui-password>` component to an element that is not a password input looks strange.
It brings confusion and moreover it brings complexity if we want to performed some transversal refactoring.